students/AurelPerianu/Lesson9/mailroom_5.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+
+class Donor:
+
+    def __init__(self, name, donations=None):
+        if ' ' in name:
+            self._first, self._last = name.split(' ')
+        else:
+            self._first = ''
+            self._last = name
+        self._name = name
+        if donations:
+            self._donations = donations
+        else:
+            self._donations = []
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def first_name(self):
+        return self._first
+
+    @property
+    def last_name(self):
+        return self._last
```

Comment:
no need to use property for something like first name or last name since there is no additional special logic.

you can access the attribute directly